ksp_plugin_adapter/flight_planner.cs

```diff
@@ -285,7 +285,7 @@ class FlightPlanner : SupervisedWindowRenderer {
           }
           var editor =
               new BurnEditor(adapter_,
-                             vessel_,
+                             vessel_!,
```

Comment:
If I understand correctly, `burn_editors_!` and `vessels_!` come from preconditions of this function; add a comment explaining that.